addons/binding/org.openhab.binding.meteostick/ESH-INF/thing/thing-types.xml

```diff
@@ -21,7 +21,10 @@
 				<options>
 					<option value="0">North America (915MHz)</option>
 					<option value="1">Europe (868MHz)</option>
-					<option value="2">Australia (915MHz)</option>
+					<option value="2">Australia</option>
```

Comment:
Why did you remove the "(915MHz)" here, while the frequency is still mentioned for NA and Europe?